client/store/pages/assessmentSection/actions/getDescriptions.ts

```diff
@@ -0,0 +1,25 @@
+import { ApiEndPoint } from '@common/api/endpoint'
+import { createAsyncThunk } from '@reduxjs/toolkit'
+import axios from 'axios'
+
+import { CountryIso } from '@meta/area'
+import { CommentableDescription } from '@meta/assessment/commentableDescription'
+
+export const getDescriptions = createAsyncThunk<
+  Record<string, Record<string, string>>,
+  { countryIso: CountryIso; assessmentName: string; cycleName: string; section: string }
+>('section/get/descriptions', async ({ countryIso, assessmentName, cycleName, section }) => {
+  const { data } = await axios.get(ApiEndPoint.Assessment.Data.descriptions(), {
+    params: { countryIso, assessmentName, cycleName, sectionName: section },
+  })
+
+  const res: Record<string, Record<string, string>> = {}
+
+  data.forEach((description: CommentableDescription) => {
```

Comment:
why not to return the data in that format from the query itself? (just jsonb_build_object)

Reply:
Resolved as: returning single content value in query and handling population in slice